key-wallet-ffi/FFI_API.md

```
@@ -1217,6 +1267,22 @@ Add a DashPay receiving funds account  # Safety - `wallet` must be a valid point
 
 ---
 
+#### `wallet_add_platform_payment_account`
+
+```c
+wallet_add_platform_payment_account(wallet: *mut FFIWallet, account_index: c_uint, key_class: c_uint,) -> crate::types::FFIAccountResult
+```
+
+**Description:**
+Add a Platform Payment account (DIP-17) to the wallet  Platform Payment accounts use the derivation path: `m/9'/coin_type'/17'/account'/key_class'/index`  # Arguments * `wallet` - Pointer to the wallet * `account_index` - The account index (hardened) in the derivation path * `key_class` - The key class (hardened) - typically 0' for main addresses  # Safety  This function dereferences a raw pointer to FFIWallet. The caller must ensure that: - The wallet pointer is either null or points to a valid FFIWallet - The FFIWallet remains valid for the duration of this call
```

markdownlint-cli2 (0.20.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)